conf/be.conf

```diff
@@ -36,15 +36,16 @@ brpc_port = 8060
 # you can specify the storage medium of each root path, HDD or SSD
 # you can add capacity limit at the end of each root path, seperate by ','
 # eg:
-# storage_root_path = /home/disk1/palo.HDD,50;/home/disk2/palo.SSD,1;/home/disk2/palo
-# /home/disk1/palo.HDD, capacity limit is 50GB, HDD;
-# /home/disk2/palo.SSD, capacity limit is 1GB, SSD;
-# /home/disk2/palo, capacity limit is disk capacity, HDD(default)
+# storage_root_path = /home/disk1/doris.HDD,50;/home/disk2/doris.SSD,1;/home/disk2/doris
+# /home/disk1/doris.HDD, capacity limit is 50GB, HDD;
+# /home/disk2/doris.SSD, capacity limit is 1GB, SSD;
+# /home/disk2/doris, capacity limit is disk capacity, HDD(default)
 # 
 # you also can specify the properties by setting '<property>:<value>', seperate by ','
 # property 'medium' has a higher priority than the extension of path
-# storage_root_path = /home/disk1/palo,medium:ssd,capacity:50
-storage_root_path = /home/disk1/palo;/home/disk2/palo
+#
+# Default value is ${DORIS_HOME}/storage, you should create it by hand.
+# storage_root_path = ${DORIS_HOME}/storage
```

Comment:
be/src/common/conf.cpp should be changed too.

Reply:
The default value in conf.cpp is already ${DORIS_HOME}/storage.